modules/mod_manager.ts

```diff
@@ -1101,10 +1173,43 @@ function GetFileWriteFunction(extension){
     else return WriteFilesToDirectory;
 }
 
+//Uses a typeguard to check if an object is or is not of type Install
+function IsCollection(arg: Install[]): Boolean {
+    if (arg.length > 1) {
+        //It is a collection
+        return true;
+    }
+    else {
+        return false
+    }
+}
+
+//Find which element in an Install[] has the desiredCollectionString as an argument
+function FindCollectionNumber(Installs: Install[], desiredCollectionString: string) : number {
+    
+    if(typeof(desiredCollectionString) == 'undefined') {
+        return 0
+    }
+
+    if(Installs.length < 2) {
+        //It has a single item
+        return 0
+    }
+    for (let i = 0; i < Installs.length; i++) {
+        const element = Installs[i];
+        if (element.itemname == desiredCollectionString) {
+            //We've found it!
+            return i;
+        }
+    }
+}
```

Comment:
I just realised this is exported, please add this to Utilities instead, as i wish to change ModManager in future to be instanced per mod.